tests/test_synced_collection.py

```diff
@@ -2,12 +2,13 @@
 # All rights reserved.
 # This software is licensed under the BSD 3-Clause License.
 import pytest
-import uuid
 import os
 import json
 from tempfile import TemporaryDirectory
 from collections.abc import MutableMapping
 from collections.abc import MutableSequence
+from hypothesis import given, strategies as st, assume
```

Comment:
I'm not sure if this is the standard `hypothesis` way of doing things, but this hurts my brain a bit since I have map `given as st` and `strategies as assume` in my head and they are on the same line. Plus now looking at it is `strategies as st`?